tests/app/user/test_rest_verify.py

```diff
@@ -249,6 +249,28 @@ def test_send_sms_code_returns_404_for_bad_input_data(client):
     assert json.loads(resp.get_data(as_text=True))['message'] == 'No result found'
 
 
+def test_send_sms_code_returns_204_when_too_many_codes_already_created(client, sample_user):
```

Comment:
might need to add more tests, if you agree with my assumptions about `expiry_datetime` and `code_used`